test/src/common/plugin/plugins-config.spec.js

```diff
@@ -132,6 +132,14 @@ describe('evaluatePluginsConfig', () => {
     pluginsConfig.kava.should.have.property('playerVersion');
     pluginsConfig.kava.playerVersion.should.equal(__VERSION__);
   });
+
+  it('should sanitize the entryName', () => {
```

Comment:
@lianbenjamin  what does this method is doing?

Reply:
it's a test- I pass an invalid payload and make sure it's sanitized and exist on the final object.